src/work-allocation/components/task-list-filter/task-list-filter.component.ts

```diff
@@ -84,7 +84,7 @@ export class TaskListFilterComponent implements OnInit, OnDestroy {
     if (this.router.getCurrentNavigation() &&
       this.router.getCurrentNavigation().extras.state &&
       this.router.getCurrentNavigation().extras.state.location) {
-      this.bookingLocations = this.router.getCurrentNavigation().extras.state.location.ids;
+      this.bookingLocations = [this.router.getCurrentNavigation().extras.state.location.id];
```

Comment:
concerned about this - does this mean multiple locations are now not being returned from the booking screens to the my work filter?

Reply:
Found one place where ids was used, so now I have changed all the other to pass an array

Reply:
[@Matei Baciu](https://hmcts-reform.slack.com/team/U03MLGQQ9RQ)
 can you liaise with 
[@Sreekanth Puligadda](https://hmcts-reform.slack.com/team/UL2H260TT)
 on the comment made by 
[@Connor McElroy](https://hmcts-reform.slack.com/team/U01DL3S8HB5)
 and look for different use cases?

Reply:
Sure, will do